pkg/controller/migmigration/stage.go

```diff
@@ -506,7 +506,7 @@ func (t *Task) ensureStagePodsDeleted() error {
 	if err != nil {
 		return liberr.Wrap(err)
 	}
-	options := k8sclient.MatchingLabels(t.PlanResources.MigPlan.GetCorrelationLabels())
+	options := k8sclient.MatchingLabels(t.stagePodIdentifierLabel())
```

Comment:
This new `stagePodIdentifier` label is used for lookup of _all_ stage pod regardless of correlation label.

Reply:
In this case we would need to filter on namespace here. When we were using the correlation label we didn't need namespace filtering for deletion since the label was enough to know it was one that this operation should delete.